Jellyfin.Server.Implementations/Users/DefaultPasswordResetProvider.cs

```diff
@@ -52,90 +63,96 @@ public class DefaultPasswordResetProvider : IPasswordResetProvider
         public bool IsEnabled => true;
 
         /// <inheritdoc />
-        public async Task<PinRedeemResult> RedeemPasswordResetPin(string pin)
+        public async Task<CodeRedeemResult> RedeemPasswordResetPin(string code, string password)
         {
-            var usersReset = new List<string>();
-            foreach (var resetFile in Directory.EnumerateFiles(_passwordResetFileBaseDir, $"{BaseResetFileName}*"))
+            SerializablePasswordReset passwordReset;
+            foreach (var file in Directory.EnumerateFiles(_passwordResetFileBaseDir, $"{BaseResetFileName}*"))
             {
-                SerializablePasswordReset spr;
-                await using (var str = File.OpenRead(resetFile))
+                using (var stream = File.OpenRead(file))
                 {
-                    spr = await _jsonSerializer.DeserializeFromStreamAsync<SerializablePasswordReset>(str).ConfigureAwait(false);
+                    passwordReset = await _jsonSerializer.DeserializeFromStreamAsync<SerializablePasswordReset>(stream).ConfigureAwait(false);
                 }
 
-                if (spr.ExpirationDate < DateTime.UtcNow)
+                if (passwordReset.ExpirationDate < DateTime.Now)
```

Comment:
Why use server time instead of UTC?

Reply:
Not sure, I've rebased this several times so it might have been a side effect because I don't remember changing this. I'll switch it back.